Zend/zend_dtrace.c

```diff
@@ -68,20 +68,22 @@ ZEND_API void dtrace_execute_ex(zend_execute_data *execute_data)
 	if (DTRACE_FUNCTION_ENTRY_ENABLED() || DTRACE_FUNCTION_RETURN_ENABLED()) {
 		classname = get_active_class_name(&scope);
 		funcname = get_active_function_name();
+		prev_funcname = get_prev_active_function_name();
+		prev_classname = get_prev_active_class_name(&scope);
```

Comment:
question: would not it be better if it had its own `scope` data ?

Reply:
You are so right, I was overriding it. Haven't noticed, it's scope is basically useless. Fixed.